run/types/DeviceWrapper.ts

```diff
@@ -1552,55 +1575,80 @@ export class DeviceWrapper implements SharedDeviceInterface {
       await sleepFor(100);
       await this.clickOnByAccessibilityID("User settings");
       await sleepFor(500);
-      await this.clickOnElementById(`network.loki.messenger:id/privacyButton`);
+      await this.clickOnElementAll(new PrivacyButton(this));
       await sleepFor(2000);
-      await this.clickOnElementAll({
-        strategy: "id",
-        selector: "android:id/summary",
-        text: "Send read receipts in one-to-one chats.",
-      });
+      await this.clickOnElementAll(new ReadReceiptsButton(this));
       await this.navigateBack(platform);
       await sleepFor(100);
       await this.navigateBack(platform);
     } else {
       await this.navigateBack(platform);
       await sleepFor(100);
       await this.clickOnByAccessibilityID("User settings");
-      await this.clickOnElementAll({ strategy: "id", selector: "Privacy" });
-      await this.clickOnElementAll({
-        strategy: "xpath",
-        selector: `//XCUIElementTypeSwitch[@name="Read Receipts, Send read receipts in one-to-one chats."]`,
-      });
+      await this.clickOnElementAll(new PrivacyButton(this));
```

Comment:
can you not make that whole if/else together now?

Reply:
yeah probably, i was just being lazy haha

Reply:
Merged